app/views/transfers/dashboards/variant-1.html

```diff
@@ -41,18 +48,19 @@ <h3 class="govuk-heading-m">
 
 <div class="govuk-width-container"><br>
   <h2 class="govuk-hint">Home</h2>
-  <h1 class="govuk-heading-xl">Academy transfer projects</h1>
+  <h1 class="govuk-heading-xl">Transfer projects (10)</h1>
 </div>
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
-    <p class="govuk-body">
-      Set up a new academy transfer project
-    </p>
-    <a href="/transfers/trust-name" role="button" draggable="false" class="govuk-button" data-module="govuk-button">
-      Start a new transfer project
-    </a>
+    <a href="/transfers/trust-name" role="button" draggable="false" class="govuk-button govuk-button--start" data-module="govuk-button">
+      Start a new project
+    <svg class="govuk-button__start-icon" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40" aria-hidden="true" focusable="false">
+    <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
+  </svg></a><br><br><br><br>
```

Comment:
Should avoid using `<br>` for spacing purposes - if you want to add margin to the top of the following content you can use the class `govuk-!-margin-top-X` - putting in the spacing unit you want. See: https://design-system.service.gov.uk/styles/spacing/ for the units

Reply:
Done.